packages/react-client/src/ReactFlightClient.js

```diff
@@ -129,17 +129,20 @@ function readRoot<T>(): T {
 }
 
 function createPendingChunk(response: Response): PendingChunk {
+  // $FlowFixMe Flow doesn't support functions as constructors
```

Comment:
looks like in a current version of Flow support for functions as constructors has gone away: [try flow](https://flow.org/try/#0MYewdgzgLgBAHjAvDMBTA7jAYiEAKASgG4AoEgMwFcxgoBLcbXGQmAbxJhhIF8g).

In the version here, it returns an inexact object but in the light that it's been already deprecated, I just added the comment that it's no longer supported.